### ISSUE TITLE
Can I use the node-gdal to create pdfs with gdal?

### ISSUE THREAD

**Issue by Charmatzis** [naturalatlas/node-gdal#177]
Hi, 
One question, I have searched the internet and I found with gdal_translate can be created pdfs from raster datasets and also gdal_translate is utility and not a native component of gdal. 
Is there a way to create pdfs with node-gdal?

THX!!!!

**Comment by BergWerkGIS** [comment]
You could iterate [`gdal.drivers`](http://naturalatlas.github.io/node-gdal/classes/gdal.html#prop-gdal.drivers) and check if PDF shows up.
If not, PDF support is not compiled into the version you are using.

**Comment by Charmatzis** [comment]
No, it doesn't show up PDF. I must use another way to print pdf through node.js....
